### PR TITLE
Update error response codes

### PR DESCRIPTION
**Pull request by benesch**
Update to include error codes from v2.2 of Kafka.

### CODE REVIEW

**Comment by benesch** [comment]
Good point! Done.

File: src-cpp/rdkafkacpp.h [hunk 1]
```diff
@@ -428,8 +428,22 @@ enum ErrorCode {
         ERR_LISTENER_NOT_FOUND = 72,
         /** Topic deletion is disabled */
         ERR_TOPIC_DELETION_DISABLED = 73,
+        /** The leader epoch is older than the broker epoch */
```
Comment (edenhill) [inline src-cpp/rdkafkacpp.h]: To be consistent with existing lingua, please remove the "The " prefixes of all these descriptions, and any the's embedded in the text too, so:
"Leader epoch is older than broker epoch"

Comment (benesch) [reply src-cpp/rdkafkacpp.h]: Ok, done.

**Comment by edenhill** [comment]
Thank you for this!

**Comment by benesch** [comment]
Thanks for reviewing! Looking forward to the next release.

**Pull request merged**